src/File.php

```diff
@@ -468,6 +468,24 @@ public function metadata(): array
         return $this->metadata;
     }
 
+    /**
+     * @param string $key
+     *
+     * @throws \RuntimeException
+     *
+     * @return mixed
+     */
+    public function metadataElement(string $key)
+    {
+        if (!isset($this->metadata[$key])) {
+            throw new RuntimeException(
+                'Metadata key does not exist'
```

Comment:
Maybe this should be nullable instead of failing this hard.
At least from a usability perspective, try catch around it all could be annoying.

Reply:
I said on Skype that you can make it nullable. :)